nidx/nidx_types/src/lib.rs

```diff
@@ -30,6 +30,16 @@ impl From<u64> for Seq {
         Self(value as i64)
     }
 }
+impl From<i32> for Seq {
```

Comment:
This is only used for tests using seq literals. I think it's better if we remove it and change the tests instead, for example to `100i64.into()` or `Seq::from(100i64)`.

Reply:
I added the implementation because I thought it might be useful but I'll remove it